cmake/CMakeCommon.cmake

```diff
@@ -100,6 +100,7 @@ cmake_path(APPEND LIBCYPHAL_ROOT "external" OUTPUT_VARIABLE CETLVAST_EXTERNAL_RO
 if (CETL_ENABLE_DEBUG_ASSERT)
     add_compile_definitions("CETL_ENABLE_DEBUG_ASSERT=1")
 else()
+    add_compile_definitions("CAVL_NO_ASSERT=1")
```

Comment:
We should be mindful that in many cases the library will be used without its CMake recipes by merely adding it to the include path list. That bypassess the logic implemented here that is aware that `CETL_ENABLE_DEBUG_ASSERT` implies no `CAVL_NO_ASSERT` and vice versa. Since we're vendoring `cavl.hpp`, should we not modify it to rely on `CETL_ENABLE_DEBUG_ASSERT` instead of `CAVL_NO_ASSERT`?

Reply:
This line is just to disable CAVL asserts for "Coverage" build flavor only.

>should we not modify it to rely on CETL_ENABLE_DEBUG_ASSERT instead of CAVL_NO_ASSERT?
Could you please clarify, are you suggesting to find/replace `CAVL_NO_ASSERT` with `CETL_DEBUG_ASSERT` at `cavl.hpp`? Thnx